test/uk/gov/hmrc/trusts/controllers/EstateVariationsControllerSpec.scala

```diff
@@ -31,56 +29,55 @@ import uk.gov.hmrc.trusts.BaseSpec
 import uk.gov.hmrc.trusts.config.AppConfig
 import uk.gov.hmrc.trusts.controllers.actions.FakeIdentifierAction
 import uk.gov.hmrc.trusts.exceptions._
-import uk.gov.hmrc.trusts.models.variation.{Variation, VariationResponse}
-import uk.gov.hmrc.trusts.services.{AuditService, DesService, ValidationService}
+import uk.gov.hmrc.trusts.models.variation.{EstateVariation, TrustVariation, VariationResponse}
+import uk.gov.hmrc.trusts.services.{AuditService, DesService}
 import uk.gov.hmrc.trusts.utils.Headers
 
 import scala.concurrent.Future
 
-class VariationsControllerSpec extends BaseSpec with BeforeAndAfter {
+class EstateVariationsControllerSpec extends BaseSpec {
 
   lazy val mockDesService: DesService = mock[DesService]
+
   lazy val mockAuditService: AuditService = mock[AuditService]
 
-  val mockAuditConnector = mock[AuditConnector]
-  val mockConfig = mock[AppConfig]
+  val mockAuditConnector: AuditConnector = mock[AuditConnector]
+  val mockConfig: AppConfig = mock[AppConfig]
 
   val auditService = new AuditService(mockAuditConnector, mockConfig)
-  val validationService = new ValidationService()
 
-  override def beforeEach() =  {
+  override def beforeEach() = {
     reset(mockDesService, mockAuditService, mockAuditConnector, mockConfig)
   }
 
-  private def variationsController = {
-    val SUT = new VariationsController(new FakeIdentifierAction(Organisation), mockConfig, mockDesService, mockAuditService, validationService)
+  private def estateVariationsController = {
+    val SUT = new EstateVariationsController(new FakeIdentifierAction(Organisation), mockDesService, mockAuditService)
     SUT
   }
 
   val tvnResponse = "XXTVN1234567890"
-  val trustVariationsAuditEvent =  "TrustVariation"
+  val estateVariationsAuditEvent =  "EstateVariation"
 
-  ".variation" should {
+  ".estateVariation" should {
 
     "not perform auditing" when {
       "the feature toggle is set to false" in {
 
-        when(mockDesService.variation(any[Variation])(any[HeaderCarrier]))
+        when(mockDesService.estateVariation(any[EstateVariation])(any[HeaderCarrier]))
           .thenReturn(Future.successful(VariationResponse(tvnResponse)))
 
         when(mockConfig.auditingEnabled).thenReturn(false)
-        when(mockConfig.variationsApiSchema).thenReturn(appConfig.variationsApiSchema)
 
-        val requestPayLoad = Json.parse(validVariationsRequestJson)
+        val requestPayLoad = Json.parse(validTrustVariationsRequestJson)
```

Comment:
should this be "validEstateVariationsRequestJson" ?

Reply:
Yes it should be. complete.